.github/workflows/merge_scalasteward_prs.yml

```diff
@@ -0,0 +1,37 @@
+name: dependency-bot
+
+on:
+  workflow_dispatch:
+  # Allows manually triggering of workflow on a selected branch via the GitHub Actions tab.
+  # GitHub blog demo: https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/.
+
+#  push:
+#    branches:
+#      - workflow
```

Comment:
can we delete commented out section?

Reply:
I think it's useful to keep this in. It's a bit of documentation for how to enable a useful feature.